pkg/e2e/verify/sccs.go

```diff
@@ -68,3 +129,145 @@ func checkSccPermissions(h *helper.H, clusterRole string, scc string) {
 	}
 	Expect(foundRule).To(BeTrue())
 }
+
+func makeMinimalSCC(name string) v1.SecurityContextConstraints {
+	scc := v1.SecurityContextConstraints{
+
+		TypeMeta: metav1.TypeMeta{
+
+			Kind:       "SecurityContextConstraints",
+			APIVersion: "security.openshift.io/v1",
+		},
+		ObjectMeta: metav1.ObjectMeta{
+
+			Name: name,
+		},
+		Groups: []string{
+			"system:authenticated",
+		},
+
+		SELinuxContext: v1.SELinuxContextStrategyOptions{
+			Type: v1.SELinuxStrategyRunAsAny,
+		},
+		RunAsUser: v1.RunAsUserStrategyOptions{
+			Type: v1.RunAsUserStrategyRunAsAny,
+		},
+		FSGroup: v1.FSGroupStrategyOptions{
+			Type: v1.FSGroupStrategyRunAsAny,
+		},
+		SupplementalGroups: v1.SupplementalGroupsStrategyOptions{
+			Type: v1.SupplementalGroupsStrategyRunAsAny,
+		},
+	}
+	return scc
+}
+
+func createScc(scc v1.SecurityContextConstraints, h *helper.H) error {
+
+	_, err := h.Security().SecurityV1().SecurityContextConstraints().Create(context.TODO(), &scc, metav1.CreateOptions{})
+	return (err)
+}
+
+func deleteScc(scc string, h *helper.H) error {
+
+	err := h.Security().SecurityV1().SecurityContextConstraints().Delete(context.TODO(), scc, metav1.DeleteOptions{})
+	return (err)
+}
+
+//Filters pods based on namespace and label
+func filterPods(namespace string, label string, h *helper.H) (*apiv1.PodList, error) {
+
+	list, err := h.Kube().CoreV1().Pods(namespace).List(context.TODO(), metav1.ListOptions{
+		LabelSelector: label})
+	if err != nil {
+		log.Printf("Could not issue create command")
+		return list, err
+	}
+
+	return list, err
+
+}
+
+//Extracts pod names from a filtered list and counts how many are in running state
+func getPodNames(namespace string, list *apiv1.PodList, h *helper.H) ([]string, int) {
+	var notReady []apiv1.Pod
+	var ready []apiv1.Pod
+	var podNames []string
+	var total int
+	var numReady int
+	notReady = nil
+	ready = nil
+	podNames = nil
+	for _, pod := range list.Items {
+		name := pod.Name
+		podNames = append(podNames, name)
+
+		phase := pod.Status.Phase
+		if phase != apiv1.PodRunning && phase != apiv1.PodSucceeded {
+			notReady = append(notReady, pod)
+		} else {
+			ready = append(ready, pod)
+
+		}
+		total = len(list.Items)
+		numReady = (total - len(notReady))
+	}
```

Comment:
```suggestion
	}
	total = len(list.Items)
	numReady = (total - len(notReady))
```

I can't see any reason to recompute these values each loop iteration. Am I missing something?